precise/iaf_psc_exp_ps.cpp

```diff
@@ -540,36 +540,75 @@ nest::iaf_psc_exp_ps::emit_instant_spike_( const Time& origin, const long lag, c
 }
 
 double
-nest::iaf_psc_exp_ps::bisectioning_( const double dt ) const
+nest::iaf_psc_exp_ps::V_m_root_function_( double t_step ) const
+{
+  const double P20 = -P_.tau_m_ / P_.c_m_ * numerics::expm1( -t_step / P_.tau_m_ );
+
+  const double P21_ex = propagator_32( P_.tau_ex_, P_.tau_m_, P_.c_m_, t_step );
+  const double P21_in = propagator_32( P_.tau_in_, P_.tau_m_, P_.c_m_, t_step );
+
+  double y2_root = P20 * ( P_.I_e_ + V_.y0_before_ ) + P21_ex * V_.y1_ex_before_ + P21_in * V_.y1_in_before_
+    + V_.y2_before_ * std::exp( -t_step / P_.tau_m_ );
+
+  return y2_root - P_.U_th_;
+}
+
+double
+nest::iaf_psc_exp_ps::regula_falsi_method_( const double dt ) const
```

Comment:
Instead of implementing this method for models that need it, could we implement it once i libnestutil? We would need to pass `V_m_root_function_` (with a better name) as a pointer-to-member-function, I think.